app/assets/javascripts/articles.js

```diff
@@ -1,13 +1,20 @@
 App.articleQueue.onIncoming = function(data) {
-  var alertDiv = $('#floating-alert');
+  var alertDiv = document.getElementById('#floating-alert');
+  var body = document.querySelectorAll('body');
 
   if (alertDiv.length === 0) {
-    alertDiv = $('<div id="floating-alert" class="alert alert-floating alert-notice">)');
-    $('body').prepend(alertDiv);
+    alertDiv = document.querySelectorAll('<div id="floating-alert" class="alert alert-floating alert-notice">');
```

Comment:
When I check the mdn documentation about `querySelectorAll` it says : 

> A DOMString containing one or more selectors to match against. This string must be a valid CSS selector string; if it's not, a SyntaxError exception is thrown.

So is this working to create an element? Shouldn't create the element using `alertDiv = document.createElement('div')` then `alertDiv.setAttribute`, etc ?

Reply:
you are correct about querySelectorAll uh what was i thinking..